Cakefile

```diff
@@ -35,6 +35,11 @@ build = (callback) ->
             console.error err
         else
             build_webpack () ->
+                jsPath = build_dir + '/APIBlueprintImporter.js'
+                js = fs.readFileSync(jsPath, 'utf-8')
+                js = js.replace('var Module={"ready"', 'var Module={"ENVIRONMENT":"WEB","ready"')
```

Comment:
I see ... hmm I have no idea how to solve it properly at the moment. If the hack is working I'd leave it with the proper comment as is and proceed further.